web/film-club-client/src/components/FilmDetailPage.js

```diff
@@ -1,54 +1,193 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { useParams } from 'react-router-dom';
-import { Box, Image, Heading, Text } from 'grommet';
+import { Box, Image, Heading, Layer, Text } from 'grommet';
+import { CirclePlay } from 'grommet-icons';
 
 import Header from './Header';
 
-const FilmDetailPage = props => {
+const FilmDetailPage = (props) => {
+	const [movieDetails, setMovieDetails] = useState({});
+	const [showVideo, setShowVideo] = useState(false);
```

Comment:
This is quite an indentation policy 😅 . Seems like some are 6 spaces, some are 8 spaces? In most projects I've seen, it's usually 2 or 4 spaces -- I'd suggest sticking with one of those and making it consistent throughout the project to prevent these deep nested indentations. If the IDE is using tabs or something, see if you can change to spaces.